src/invoke_training/training/finetune_lora/finetune_lora_sdxl.py

```diff
@@ -432,16 +439,15 @@ def compute_time_ids(original_size, crops_coords_top_left):
     unet_conditions = {"time_ids": add_time_ids}
 
     # Get the text embedding for conditioning.
-    # The text_encoder_output may have been cached and included in the data_batch. If not, we calculate it here.
-    all_prompt_embeds = data_batch.get("text_encoder_output", None)
-    if all_prompt_embeds is None:
+    # The text encoder output may have been cached and included in the data_batch. If not, we calculate it here.
+    if "prompt_embeds" in data_batch:
+        prompt_embeds = data_batch["prompt_embeds"]
+        pooled_prompt_embeds = data_batch["pooled_prompt_embeds"]
```

Comment:
I assume this will always exist when prompt_embeds does

Reply:
Correct. If `"pooled_prompt_embeds"` wasn't set we'd want to raise an Exception.